tsconfig.json

```diff
@@ -24,7 +24,8 @@
 		"inlineSourceMap": false,
 
 		// watch=true breaks CI build steps
-		"watch": false
+		"watch": false,
+		"newLine": "lf"
```

Comment:
Why do we need to set newLine to LF? Isn't `git` doing the crlf -> lf conversion?